scipy/integrate/_quadpack_py.py

```diff
@@ -281,6 +281,89 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
         ``infodict['rslist']``.  See the explanation dictionary (last entry
         in the output tuple) for the meaning of the codes.
 
+
+    **Details of QUADPACK level routines**
+
+    *quad* calls routines from the FORTRAN library QUADPACK. This section
+    provides details on the conditions for each routine to be called and a
+    short description of each routine. The routine called depends on
+    ``weight``, ``points`` and the integration limits ``a`` and ``b``.
```

Comment:
Functions and arguments from the current function should use single backticks. We use double backticks for short snippets of codes. Here and bellow it should be adjusted.
```suggestion
    `quad` calls routines from the FORTRAN library QUADPACK. This section
    provides details on the conditions for each routine to be called and a
    short description of each routine. The routine called depends on
    `weight`, `points` and the integration limits `a` and `b`.
```

Reply:
Done. There were a few further down too that are now also done.